### PR TITLE
Fix/disable group posting

### PR DESCRIPTION
**Pull request by melisabok**
A few changes needed for ECCV:

- `get_result` doesn't post **all the paper groups** anymore. `create_blind_submissions` does this job now. If the conference is single blind then `conference.create_paper_groups` must be called. 
- read 'contac't field from the configuration note.
- given blind submissions have the same number that the original I improved the way we create the blind  submissions, without multiple posts.
- `setup_matching` builds the conflicts **on demand** only, this is to avoid the remove/create of all the edges that can be expensive.
- `helpers` is reading a new property from the configuration note `remove_submission_options` to remove default fields from the form.
- Fix duedate for expertise and decision invitations.
- Post conflict edges in each paper iteration. Don't wait until the end of the loop to post all the edges.
- When building the scores check if a paper was already deleted. 
- Fixes https://github.com/openreview/openreview-py/issues/592
- Fix tests.

This PR will help us to use the configuration note of ECCV to setup all the other stages.

### CODE REVIEW

File: openreview/conference/builder.py [hunk 1]
```diff
@@ -473,6 +474,48 @@ def create_desk_reject_invitations(self, reveal_authors=False, reveal_submission
...
+    def create_paper_groups(self, authors=False, reviewers=False, area_chairs=False):
+
+        notes_iterator = self.get_submissions(sort='number:asc', details='original')
+        author_group_ids = []
+
+        for n in notes_iterator:
+            # Paper group
+            group = self.__create_group(
+                group_id = '{conference_id}/Paper{number}'.format(conference_id = self.id, number = n.number),
+                group_owner_id = self.get_area_chairs_id(number = n.number) if self.use_area_chairs else self.id,
```
Comment (carlosmondra) [inline openreview/conference/builder.py]: Just removing some spaces for convention.
```suggestion
                group_id = '{conference_id}/Paper{number}'.format(conference_id=self.id, number=n.number),
                group_owner_id = self.get_area_chairs_id(number=n.number) if self.use_area_chairs else self.id,
```

File: openreview/conference/builder.py [hunk 2]
```diff
@@ -473,6 +474,48 @@ def create_desk_reject_invitations(self, reveal_authors=False, reveal_submission
...
+                if n.details and n.details.get('original'):
+                    authorids = n.details['original']['content']['authorids']
+                author_paper_group = self.__create_group(self.get_authors_id(n.number), self.id, authorids)
+                author_group_ids.append(author_paper_group.id)
+
+            # Reviewers Paper group
+            if reviewers:
+                self.__create_group(
+                    self.get_reviewers_id(number = n.number),
+                    self.get_area_chairs_id(number = n.number) if self.use_area_chairs else self.id,
```
Comment (carlosmondra) [inline openreview/conference/builder.py]:
```suggestion
                    self.get_reviewers_id(number=n.number),
                    self.get_area_chairs_id(number=n.number) if self.use_area_chairs else self.id,
```

File: openreview/conference/builder.py [hunk 3]
```diff
@@ -473,6 +474,48 @@ def create_desk_reject_invitations(self, reveal_authors=False, reveal_submission
...
+                    is_signatory = False)
+
+                # Reviewers Submitted Paper group
+                self.__create_group(
+                    self.get_reviewers_id(number = n.number) + '/Submitted',
+                    self.get_area_chairs_id(number = n.number) if self.use_area_chairs else self.id,
```
Comment (carlosmondra) [inline openreview/conference/builder.py]:
```suggestion
                    self.get_reviewers_id(number=n.number) + '/Submitted',
                    self.get_area_chairs_id(number=n.number) if self.use_area_chairs else self.id,
```

File: openreview/conference/builder.py (hunk 2 above)
Comment (melisabok) [reply openreview/conference/builder.py]: Thanks, it is time to start following the python best practices.

File: openreview/conference/builder.py [hunk 4]
```diff
@@ -473,6 +474,48 @@ def create_desk_reject_invitations(self, reveal_authors=False, reveal_submission
...
+                    is_signatory = False)
+
+            # Area Chait Paper group
+            if self.use_area_chairs and area_chairs:
+                self.__create_group(self.get_area_chairs_id(number = n.number), self.id)
```
Comment (carlosmondra) [inline openreview/conference/builder.py]:
```suggestion
                self.__create_group(self.get_area_chairs_id(number=n.number), self.id)
```

File: openreview/conference/builder.py [hunk 5]
```diff
@@ -489,42 +532,43 @@ def create_blind_submissions(self, force=False, hide_fields=[]):
         for note in tools.iterget_notes(self.client, invitation = self.get_submission_id(), sort = 'number:asc'):
             blind_note = submissions_by_original.get(note.id)
             if not blind_note:
+
+                blind_content = {
+                    'authors': ['Anonymous'],
+                    'authorids': [self.get_authors_id(number = note.number)],
```
Comment (carlosmondra) [inline openreview/conference/builder.py]:
```suggestion
                    'authorids': [self.get_authors_id(number=note.number)],
```

File: openreview/conference/invitation.py [hunk 6]
```diff
@@ -769,7 +769,7 @@ def __init__(self, conference, name, review, start_date, due_date, additional_fi
                 super = review.invitation,
                 cdate = tools.datetime_millis(start_date),
                 duedate = tools.datetime_millis(due_date),
-                expdate = tools.datetime_millis(due_date + datetime.timedelta(minutes = LONG_BUFFER_DAYS)) if due_date else None,
+                expdate = tools.datetime_millis(due_date + datetime.timedelta(days = LONG_BUFFER_DAYS)) if due_date else None,
```
Comment (mohituniyal) [inline openreview/conference/invitation.py]: ah, good catch :)

**Pull request merged**